autolog-ui/src/components/InventoryDropdown/InventoryDropdown.jsx

```diff
@@ -49,21 +82,51 @@ export default function InventoryDropdown() {
                         <img src={open ? arrowCollapse : arrowExpand} />
                     </div>
                 </div>
-                {/* items wrapper */}
+                {/* 
+                    Inventory List 
+                */}
                 {open && (
-                    <ul className="dropdown-inventory-list">
-                        {/* items */}
-                        {accessibleInventories?.map((item, idx) => {
-                            return (<div key={idx} className={`dropdown-inventory-item ${(idx === (accessibleInventories?.length - 1) ? 'last' : '')}`} onClick={() => handleOnClick(item)}>
-                                <span>{_.capitalize(item?.inventoryName)}</span>
-                            </div>)
-                        })}
+                    <>
+                        <DropdownOverlay onBlur={toggle}/>
+                        <ul className="dropdown-inventory-list">
+                            {/* items */}
+                            {accessibleInventories?.map((item, idx) => {
+                                return (<div key={idx} className={`dropdown-inventory-item`} onClick={() => handleOnClick(item)}>
+                                    <span>{_.capitalize(item?.inventoryName)}</span>
+                                </div>)
+                            })}
+
+                            {/* button that leads to create inventory */}
+                            <div className={`dropdown-inventory-item button-create-inventory last`} onClick={toggleModalOnDropdown}>
+                                <span>Create New Inventory</span>
+                            </div>
                     </ul>
+                    </>
                 )}
+
+                {/* 
+                    Modal for new inventory
+                */}
+
+                <AnimatePresence
+                    initial={false}
+                    exitBeforeEnter={true}
+                    onExitComplete={() => null}>
+                    {
+                        modalOpen && 
+                        <Modal
+                            title={'Create New Inventory'}
+                            body={<Form formState={formState} setFormState={setFormState} formArray={formArray} />}
+                            onSubmit={() => console.log("create")}
```

Comment:
Still here

Reply:
Sorry I was trying to get through a little problem with the modal itself before doing any other commits to not pile up bugs

Reply:
I added an API call instead of console logging now 👍🏻